lms/static/sass/views/_login-register.scss

```diff
@@ -350,6 +350,8 @@ $sm-btn-linkedin: #0077b5;
         @extend %btn-secondary-grey-outline;
         width: 130px;
         padding: 0 0 0 ($baseline*2);
+        margin-right: 6px;
```

Comment:
@Kelketek is correct, for anything that can be a derivative of our ``$baseline`` Sass spacing/layout scale variable (~20px), please reference it. In this case, something like ``($baseline/4)`` would be best.